test/python/WMCore_t/MicroService_t/MicroService_t.py

```diff
@@ -7,10 +7,11 @@
 from __future__ import division, print_function
 
 import json
-import cherrypy
 import unittest
 
+import cherrypy
```

Comment:
I'm not sure why do you need cherrypy here since all applied changes do not use cherrypy.

Reply:
Don't spend time looking at test/* changes (and/or pylint changes). As you can see, this is just an automatic reformat of this module.